tests/tests/test-nonce.ts

```diff
@@ -35,4 +36,47 @@ describeWithMoonbeam("Moonbeam RPC (Nonce)", `simple-specs.json`, (context) => {
     expect(await context.web3.eth.getTransactionCount(GENESIS_ACCOUNT, "pending")).to.eq(1);
     expect(await context.web3.eth.getTransactionCount(GENESIS_ACCOUNT, "earliest")).to.eq(0);
   });
+
+  step("nonce should not be reset to 0 when emptying dust accounts", async function () {
```

Comment:
`step` is only necessary when step will have some kind of dependence on previous tests ( a bad thing to do but makes the tests faster (it was at least)
`it` should be used instead